src/App.tsx

```diff
@@ -2,13 +2,23 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { PostList } from './Components/PostList/PostList';
+
+const readyPost = posts.map(post => {
+  return {
+    ...post,
+    user: users.find(user => user.id === post.id),
+    comments: comments.filter(comment => comment.postId === post.id),
+  };
+});
```

Comment:
It can be shorter ;)
```suggestion
const readyPost = posts.map(post => ({
  ...post,
  user: users.find(user => user.id === post.id),
  comments: comments.filter(comment => comment.postId === post.id),
}));
```